css/properties/flex-flow.json

```diff
@@ -0,0 +1,73 @@
+{
+  "css": {
+    "properties": {
+      "flex-flow": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/flex-flow",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "29"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "21"
+              }
+            ],
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "28"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "49",
+                "notes": "For Firefox 48, the <code>-webkit-</code> prefix was available with the <code>layout.css.prefixes.webkit</code> preference. Starting with Firefox 49, the preference defaults to <code>true</code>."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "28"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "49",
+                "notes": "For Firefox 48, the <code>-webkit-</code> prefix was available with the <code>layout.css.prefixes.webkit</code> preference. Starting with Firefox 49, the preference defaults to <code>true</code>."
+              }
+            ],
+            "ie": {
+              "version_added": "11"
+            },
+            "ie_mobile": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "12.10"
+            },
+            "opera_android": {
+              "version_added": "12.10"
+            },
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "6.1"
+            },
+            "safari_ios": {
```

Comment:
See flex-basis comment there.

Reply:
Wait, this one is different. This one is prefixed and a different version number. Your `flex-basis` comment called for changing the version number for the unprefixed property. What change should I be making here?

Reply:
The ios (and desktop) flex-flow property has been unprefixed at the same time as the flex-basis one, so like for the other one, you have to add an entry for the unprefix version, in addition to the prefixed entry.